package-lock.json

```diff
@@ -1,8380 +1,8 @@
 {
-  "name": "jmap-client-ts",
```

Comment:
Since you did not modify package.json file, you should not include this file in the pull request.

Also if you use the latest npm version, you can have `"lockfileVersion" : 2` instead of `1`

Reply:
package-lock.json is clearly noise - I support removing this timely as long as it is done in a separate commit.

Reply:
I think this was discussed earlier and it was concluded that it should be committed to the repository. What to other frontend devs think about this. Should it stay?

Reply:
I think it depends on 2 things:
1. Are we using definitive dependency versions?
1.1. Relative: `"some-dependency": "^1.0.0"` or `"some-dependency": "~1.0.0"`
1.2. Concrete: `"some-dependency": "1.0.0"`
2. Is there `.npmrc` file preset and does it request some dependencies from locations other than npmjs.org. If so, then it might introduce some unexpected result for other library users.

Reply:
> package-lock.json is clearly noise - I support removing this timely as long as it is done in a separate commit.

Well, we discussed about it and I chose to not remove it.
I was just mentioning that since the `package.json` file was not modified, there was no reason to include package-lock changes on this pull request.
And I saw some references to the fork of ee-email on it, so I supposed there was a problem with it.